core/templates/dev/head/pages/exploration_player/AnswerClassificationServiceSpec.js

```diff
@@ -336,28 +349,18 @@ describe('Answer classification service with string classifier enabled',
       }
     };
 
-    it('should query the backend if no answer group matches and interaction ' +
-       'is trainable', function() {
-      var backendClassifiedOutcome = {
-        outcome: 'outcome',
-        answer_group_index: 0,
-        rule_spec_index: 0,
-        classification_categorization: EXPLICIT_CLASSIFICATION
-      };
+    it('should query the prediction service if no answer group matches and ' +
```

Comment:
@prasanna08 Ahh. Thats fine. I wasn't very sure about what gets returned when we use the ML classifier.